Bukkit/src/main/java/com/plotsquared/bukkit/entity/ReplicatingEntityWrapper.java

```diff
@@ -152,16 +152,16 @@ public ReplicatingEntityWrapper(Entity entity, short depth) {
                 this.stack = item.getItemStack();
                 return;
             case "ITEM_FRAME":
-                this.x = Math.floor(this.x);
-                this.y = Math.floor(this.y);
+                this.x = Math.floor(this.getX());
+                this.y = Math.floor(this.getY());
                 this.z = Math.floor(this.z);
                 ItemFrame itemFrame = (ItemFrame) entity;
                 this.dataByte = getOrdinal(Rotation.values(), itemFrame.getRotation());
                 this.stack = itemFrame.getItem().clone();
                 return;
             case "PAINTING":
-                this.x = Math.floor(this.x);
-                this.y = Math.floor(this.y);
+                this.x = Math.floor(this.getX());
+                this.y = Math.floor(this.getY());
                 this.z = Math.floor(this.z);
```

Comment:
*

Reply:
This is intentional. I cleaned up all coordinate access shit everywhere, so we use getters for everything but assignment